### ISSUE TITLE
Recieving "Binaries missing/corrupted." Error, even with antivirus disabled.

### ISSUE THREAD

**Issue by NebulasStars** [StefanLobbenmeier/youtube-dl-gui#92]
I disabled Windows Defender, and told Malwarebytes to ignore the program and it's behaviour, and still I receive the error. I do not know how to replicate this bug, since it's been happening since I downloaded it. I expected for disabling the antivirus to make the program start to actually download videos, but I continue to see "Binaries Missing/Corrupted."
![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/72106408/3cdf1c5d-a9eb-4369-baa2-261a585e7bb9)
I am on Windows 11, using version 2.4.0, having installed directly from Github.

**Comment by StefanLobbenmeier** [comment]
I think you are ignoring the wrong file here. Can you try ignoring the yt-dlp.exe in your app data?

Should be something like %APPDATA%/openvideodownloader/binaries, let me check

**Comment by StefanLobbenmeier** [comment]
Can you also check the antivirus history? There should be an entry that either blocked or removed in a file in the folder above

**Comment by NebulasStars** [comment]
Hi! I found the file by tracing back my shortcut. With Malwarebytes, ignoring one folder will ignore all subfolders and any items inside, so it can't be that I didn't also ignore a subfolder or an .exe file :(
![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/72106408/0fe1c0d3-25e6-4edc-b688-7291526cafa5)
On top of this, no files related to the program have been quarantined, and my MalwareBytes history is empty
![image](https://github.com/StefanLobbenmeier/youtube-dl-gui/assets/72106408/461dfde1-33d4-45ea-ba08-e23d6f57a527)

**Comment by NebulasStars** [comment]
Also, I unfortunately don't know how to manually roll back to a previous version, as I used the executable installer and I'm not that savvy with manually changing programs.

**Comment by StefanLobbenmeier** [comment]
Can you also check the Windows Defender Protection History?

https://support.microsoft.com/en-us/windows/protection-history-f1e5fd95-09b4-46d1-b8c7-1059a1e09708

**Comment by Zerem-12** [comment]
Yeah I'm suffering the same issue too even with cookies downloaded

**Comment by Zerem-12** [comment]
Nevermind this is one year old lmfao
